### PR TITLE
Fix npm script to run prettier

### PR DESCRIPTION
**Pull request by 0xaptosj**
Previously I was getting prettier not found error if i don't have prettier installed globally.

### CODE REVIEW

File: templates/fungible-asset-template/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
-    "_fmt": "prettier 'frontend/**/*.ts'",
+    "_fmt": "npx prettier 'frontend/**/*.ts'",
```
Comment (0xmaayan) [inline templates/fungible-asset-template/package.json:15]: i think it is because you need to first run "npm i"

Comment (0xaptosj) [reply templates/fungible-asset-template/package.json:15]: i have run "npm i", but i think if you just run prettier without npx, that requires you to have prettier globally

Comment (0xmaayan) [reply templates/fungible-asset-template/package.json:15]: Interesting.. i dont have it globally and was able to run it with npm on this env

Comment (0xaptosj) [reply templates/fungible-asset-template/package.json:15]: No issue with `pnpm fmt` in sdk repo, maybe it's a npm thing?

Comment (0xaptosj) [reply templates/fungible-asset-template/package.json:15]: ill close the PR for now, can fix it if report by others

**Pull request closed without merging**